builder/remote.go

```diff
@@ -65,6 +66,14 @@ func MakeRemoteContext(remoteURL string, contentTypeHandlers map[string]func(io.
 	return MakeTarSumContext(contextReader)
 }
 
+// DefaultDownloadFallbackContext creates a context with a progress reader as output.
+// This is used as default for unknown mime types, like the ones we get downloading tarballs.
+func DefaultDownloadFallbackContext(remoteURL string, writer io.Writer, size int64) func(io.ReadCloser) (io.ReadCloser, error) {
```

Comment:
Do we really need this function? It's super unclear what it's doing. There is no word context in it(because I think it's not context). It's impossible to understand how to use it. Maybe it shouldn't be separate func.